### DISCUSSION TITLE
实话讲有些内容有点扯，真假还得自己判断

### DISCUSSION THREAD

**Discussion by AZWzzz** [The-Run-Philosophy-Organization/run#486] [General]
新疆那个纯纯谣言了

**Comment by MarionNoel** [comment]
润太祖可是特大人命事故入狱的局长公子，贪二代，有钱的大老板。他大发善心，关爱新一代it耗材，安排咱们世世代代跟着他，给他做贡献。
https://github.com/MarionNoel/juzhanggongzi/discussions/
